AndroidSignalRJavaClientSample/app/build.gradle

```diff
@@ -0,0 +1,33 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.example.mikaelmengistu.androidsignalrjavaclientsample"
+        minSdkVersion 24
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        targetCompatibility 1.8
+        sourceCompatibility 1.8
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation 'com.android.support:appcompat-v7:28.0.0-rc01'
+    implementation 'com.android.support.constraint:constraint-layout:1.1.2'
```

Comment:
I think we probably still need to register the OSS components our samples use (cc @Eilon)

Reply:
Yes. I assume for these they are just "package references" (like NuGet), so it's just the *build* process that downloads them. We don't actually check in the OSS components, right? (Regardless of binary vs. source.)

Reply:
> We don't actually check in the OSS components, right? 

That is correct with the **exception** of the Gradle Wrapper. We already have that registered as "Distribution" from use in aspnet/SignalR though. Gradle has a binary "wrapper" component that serves as a bootstrapper which we are supposed to check in.

Reply:
Yup I saw that Gradle change.